### PR TITLE
Update Node LTS version to latest 10

### PR DESCRIPTION
**Pull request by gpcastro**
We're currently defaulting Node.Js version to 8.11.2

### CODE REVIEW

File: src/BuildScriptGenerator/Node/NodeScriptGeneratorOptionsSetup.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ internal class NodeScriptGeneratorOptionsSetup : IConfigureOptions<NodeScriptGen
         internal const string NodeSupportedVersionsEnvVariable = "NODE_SUPPORTED_VERSIONS";
         internal const string NpmSupportedVersionsEnvVariable = "NPM_SUPPORTED_VERSIONS";
         internal const string LegacyZipNodeModules = "ENABLE_NODE_MODULES_ZIP";
-        internal const string NodeLtsVersion = "8.11.2";
+        internal const string NodeLtsVersion = NodeVersions.Node10Version;
```
Comment (dorfire) [inline src/BuildScriptGenerator/Node/NodeScriptGeneratorOptionsSetup.cs]: should this also move to the constants file?

File: tests/Oryx.BuildImage.Tests/VersionInformationTest.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 using System;
 using Microsoft.Oryx.BuildScriptGenerator.DotNetCore;
+using Microsoft.Oryx.BuildScriptGenerator.Node;
```
Comment (dorfire) [inline tests/Oryx.BuildImage.Tests/VersionInformationTest.cs]: ?

Comment (gpcastro) [reply tests/Oryx.BuildImage.Tests/VersionInformationTest.cs]: I've moved `NodeConstants` from test project to this namespace

File: src/BuildScriptGenerator/Node/NodeScriptGeneratorOptionsSetup.cs (hunk 1 above)
Comment (gpcastro) [reply src/BuildScriptGenerator/Node/NodeScriptGeneratorOptionsSetup.cs]: Makes sense

**Pull request merged**